app/models/follow.rb

```diff
@@ -0,0 +1,4 @@
+class Follow < ApplicationRecord
+    belongs_to :follower, class_name: 'User'
+    belongs_to :followee, class_name: 'User'
```

Comment:
You are missing the `foreign_key` since your fields are called  `:follower_user_id` `:followee_user_id` the way you have it would cause to looked for `follower_id` `followee_id` columns and is going to lead to errors

```
3.1.2 :011 > f = Follow.new follower_user_id: u2.id, followee_user_id: u.id
 => #<Follow:0x00007fb0a2fa0870 id: nil, follower_user_id: 1, followee_user_id: 2, created_at: nil, updated_at: nil> 
3.1.2 :012 > f.valid?
 => false 
3.1.2 :013 > f.errors
 => #<ActiveModel::Errors [#<ActiveModel::Error attribute=follower, type=blank, options={:message=>:required}>, #<ActiveModel::Error attribute=followee, type=blank, options={:message=>:required}>]> 
3.1.2 :014 > f.errors.messages
 => {:follower=>["must exist"], :followee=>["must exist"]} 

```